pkg/querier/querier.go

```diff
@@ -177,7 +177,7 @@ func (q *SingleTenantQuerier) SelectSamples(ctx context.Context, params logql.Se
 
 	params.SampleQueryRequest.Deletes, err = q.deletesForUser(ctx, params.Start, params.End)
 	if err != nil {
-		return nil, err
+		level.Error(spanlogger.FromContext(ctx)).Log("msg", "failed loading deletes for user", "err", err)
```

Comment:
nit: since you saying `for user`, nice to include `tenat-id` as well in the error log?

Reply:
That should be covered by the spanlogger: https://github.com/grafana/dskit/blob/ebb5c6de233d26951aef41eada3cab3295ecb368/spanlogger/spanlogger.go#L99-L111